src/instructlab/lab.py

```diff
@@ -1103,12 +1110,13 @@ def train(
 
         convert_llama_to_gguf(model=final_results_dir, pad_vocab=True)
 
-        gguf_models_dir = "./models"
-        if not os.path.isdir(gguf_models_dir):
-            os.mkdir(gguf_models_dir)
-        shutil.copy(final_results_dir + "/ggml-model-f16.gguf", gguf_models_dir)
-        # cleanup original copy of model
-        os.remove(final_results_dir + "/ggml-model-f16.gguf")
+        # Remove safetensors files to save space, were done with them here
+        # and the huggingface lib has them cached
+        for file in glob(final_results_dir + "/*.safetensors"):
+            os.remove(file)
+
+        shutil.move(final_results_dir + "/ggml-model-f16.gguf", gguf_models_file)
```

Comment:
only if you have to make another change anyway, would be good to make this `os.path.join()`. I know this isn't new code though, so it's not worth blocking over